plugins/rplanners/parabolicsmoother2.cpp

```diff
@@ -12,6 +12,7 @@
 // You should have received a copy of the GNU Lesser General Public License along with this program.
 // If not, see <http://www.gnu.org/licenses/>.
 #include "openraveplugindefs.h"
+#include <cfloat>
```

Comment:
@undisputed-seraphim why is cfloat necessary?

Reply:
I'm using gcc. I was trying to build OpenRAVE in native Debian 11 (without Docker).

Reply:
@rdiankov Compiling with the cmake flags above results in an error from gcc stating that DBL_MAX was not defined anywhere.

Reply:
let me check it thanks

Reply:
There is an include chain:

- plugins/include/openraveplugindefs.h
- include/openrave/openrave.h
- include/openrave/collisionchecker.h
- include/openrave/openravejson.h
- /usr/include/boost/lexical_cast.hpp
- /usr/include/boost/math/special_functions/sign.hpp
- /usr/include/boost/math/tools/config.hpp
- cfloat
- float.h

So it depends on Boost version.

@undisputed-seraphim For reference, may I ask what boost version you used? Note that Debian 11's stock is 1.74.

Reply:
I mean, DBL_MAX issue does not reproduce by using Boost 1.74.

Reply:
> So it depends on Boost version.

and adding cfloat include is correct, I approve. I clarify that

Reply:
He used Debian 11 stock, have no idea :joy: